api/v1alpha1/clustergroupupgrade_types.go

```diff
@@ -101,7 +101,7 @@ type ClusterGroupUpgradeSpec struct {
 	// Once set to true, the clusters start being upgraded, one batch at a time.
```

Comment:
It seems nothing has changed in this file except the formatting (automatically applied, I presume). Or was it "tabs to spaces" kind of thing?
Should we exclude this file from the commit?

Reply:
when i launched hack/golangci-lint.sh it was failing with a gofmt error, thats why i had to fix it. I wonder why it was not detected before.